python/format/src/datasets.py

```diff
@@ -0,0 +1,55 @@
+import dataclasses
+import json
+from typing import Union
+
+from absl import logging
+from etils import epath
+
+from format.src import errors
+from format.src import graphs
+
+FileOrFilePath = Union[str, epath.PathLike, dict]
+
+
+def _load_file(file: FileOrFilePath) -> dict:
+    if isinstance(file, str):
+        file = epath.Path(file)
+        if not file.exists():
+            raise ValueError(f"File {file} does not exist.")
+    if isinstance(file, epath.PathLike):
+        with file.open() as file:
+            file = json.load(file)
+    if not isinstance(file, dict):
+        raise ValueError("The file is not a valid JSON-LD, because it's not a dict.")
```

Comment:
Nit: Being a "dict" sounds like a Python specific attribute. In JSON terminology, should that be an "object" instead?